src/locale/en-au.js

```diff
@@ -0,0 +1,17 @@
+import dayjs from 'dayjs'
+
+const locale = {
+  name: 'en-au',
+  weekdays: 'Sunday_Monday_Tuesday_Wednesday_Thursday_Friday_Saturday'.split('_'),
+  months: 'January_February_March_April_May_June_July_August_September_October_November_December'.split('_'),
+  weekStart: 1,
```

Comment:
I do not believe this is correct (or else Moment has changed it since you made this PR). [According to Moment](https://github.com/moment/moment/blob/master/locale/en-au.js) Australia has a weekStart of 0 (Sunday). I can't find a canonical source to back up either choice, I wanted to point out that Day.js is deviating from Moment in this case. When tested on MacOS (System Preferences -> Languages & Region -> change Region to Australia) it appears that Australia starts their week on a Sunday.